### DISCUSSION TITLE
How to stipulate 0 arguments?

### DISCUSSION THREAD

**Discussion by jordan-da** [urfave/cli#1913] [Q&A]
I have multiple leafy subcommands that shouldn't take any arguments, is there a built in native way to stipulate that a command should take no args?

I understand that I could add an action to check for this, i was looking for something built in... for example, I totally assumed that if you provided an empty list of `Arguments` it would inform the library that we don't want any args:

```golang
		Arguments:   []cli.Argument{},
```

**Comment by dearchap** [comment]
@jordan-da which release of urfave/cli are you using ?

**Reply by jordan-da** [reply]
we are currently trying alpha now but wouldn't be opposed to switch back
